extensions/amp-lightbox/1.0/component.jss.js

```diff
@@ -38,39 +38,61 @@ before allowing buttons to be activated. */
 };
 
 const wrapper = {
-  top: 0,
-  bottom: 0,
-  left: 0,
-  right: 0,
-  width: '100%',
-  height: '100%',
-  position: 'fixed',
-  boxSizing: 'border-box',
-};
+  top: '0 !important',
+  left: '0 !important',
+  bottom: '0 !important',
+  right: '0 !important',
+  width: '100% !important',
+  height: '100% !important',
+  position: 'fixed !important',
+  boxSizing: 'border-box !important',
+
+  /** Scroll containment:
+   *  overflow: hidden does not trigger overscrollBehavior, so
+   *  overflow: scroll is applied even though the lightbox should not scroll.
+   *  We rely instead on the overflow: hidden in content layer to prevent
+   *  scrolling inside the lightbox.
+   */
+  overflow: 'scroll !important', // Prevent scrolling inside lightbox.
+  overscrollBehavior: 'none !important', // Prevent scrolling outside lightbox.
 
-// User overridable styles
-const defaultStyles = {
+  // User overridable styles
   zIndex: 1000,
   backgroundColor: 'rgba(0, 0, 0, 0.9)',
   color: '#fff',
 };
 
-/* 
-  overflow: hidden does not trigger overscrollBehavior, so 
-  overflow: auto is applied even though the lightbox should not scroll.
-  We rely instead on the overflow: hidden in ContainWrapper to prevent 
-  scrolling inside the lightbox.
-*/
-const containScroll = {
-  overflow: 'auto', // Prevent scrolling inside lightbox.
-  overscrollBehavior: 'none', // Prevent scrolling outside lightbox.
+const scrollable = {};
+const content = {
+  overflow: 'hidden !important',
+  '&$scrollable': {
+    overflow: 'scroll !important',
+  },
+};
+
+const backdrop = {
+  position: 'fixed !important',
+  top: '0 !important',
+  left: '0 !important',
+  width: '101vw !important',
```

Comment:
I'm assuming the `101vw` is to hide the scrollbar by making it off screen in a similar way to `sidebar`'s implementation?  Just wondering if `101vw` is wide enough?

Reply:
I don't see a scrollbar in my manual tests, can you provide instructions for how to test for this?

Reply:
Sure, for sidebar the weird overscroll behavior did not show up in Storybook.  It does show up if you create your own example amp page.  I'll assume you're already doing that in your manual tests.

Then, you can hover on the backdrop and scroll up and down to see (or not see, since it's off the end of the page) the scrollbar.

For lightbox, looks like the actual Lightbox modal takes up the entire viewport so you can't really hover on the backdrop.  But the idea is we want to get the backdrop to scroll up and down.  I guess one test you can do is give the lightbox modal enough content so that it can scroll up and down.  And since it's slightly transparent (or you can adjust the opacity to make it more transparent), to see the backdrop.  Then scroll up and down on the lightbox modal, and this should trigger scrolling on the backdrop (due to the odd overscroll behavior).  When the backdrop is scrolling, you can just confirm that you don't see the scrollbar on the right.  If you don't, then everything should be good!

Reply:
Yes, another option is to press the `Open canvas in new tab` button in Storybook, the one in the middle here: 
![image](https://user-images.githubusercontent.com/10456171/110353663-5093fe00-8005-11eb-87c8-e95e0a0ff528.png)

Here is [the corresponding example](https://storage.googleapis.com/amp-test-website-1/amp_nomodule_65b21669ab508e175e6e68ba7c3de9e73f654eea/examples/storybook/amp/iframe.html?id=amp-lightbox-1-0--scrollable) from the PR deploy bot generated Storybook. I'm not seeing the scrollbar from the backdrop -- there is a scrollbar on the scrollable content, which is intended. Does this look right to you?

Reply:
Thanks Caroline.  Testing manually everything seems to be working fine on the link you gave.  However, I was a bit confused, I was not able to locate the two new "backdrop" `div's` you added with this PR.  

They should be before the `</c>` right?  Am I missing something?

![image](https://user-images.githubusercontent.com/25626770/110392291-b51b8100-8036-11eb-9b3c-d9a1e73e6ebd.png)

Reply:
Sorry about the link - looks like it was using the cdn.ampproject.org extension rather than the storage.google.apis one. Thanks for being so thorough and finding it.

There's a better demo coming in this next PR deploy, at examples/amp-lightbox.amp.html. Please take a look at that one.